server/http_endpoints_test.go

```diff
@@ -27,6 +28,66 @@ import (
 
 const testdataPath = "testdata/endpoints"
 
+func TestBackend_BackendVariable(t *testing.T) {
```

Comment:
Add test with accessing `backend_request`/`backend_response` in other contexts, like `set_response_headers`.